framework/src/iotbus/iotbus_spi.c

```diff
@@ -75,44 +74,50 @@ iotbus_spi_context_h iotbus_spi_open(unsigned int bus, const struct iotbus_spi_c
 		return NULL;
 	}
 
+    /*  Open driver */
+	char gpio_dev[16] = { 0, };
+	snprintf(gpio_dev, 16, "/dev/spi-%d", bus);
+	int fd = open(gpio_dev, O_RDWR);
```

Comment:
some of operation takes long long time thorugh the spi, (c.f write operation of file system) so each of flash provides register to check flash's state.
But if user use spi bus with port which initiated for flash, it will write very wrongly.
How about add some description of api to avoid use iotbus to control flash??

Reply:
That's good idea. so if some system module(like file system) use spi and user use it then it would cause the issue. I'll add the warning message ahead of function declaration in iotbus header file.